apis/v1alpha1/httproute_types.go

```diff
@@ -377,70 +396,116 @@ type HTTPRequestHeaderFilter struct {
 	//   GET /foo HTTP/1.1
 	//   My-Header2: DEF
 	//
-	// Support: extended?
+	// Support: Extended
+	// +kubebuilder:validation:MaxItems=16
 	Remove []string `json:"remove"`
-
-	// TODO
 }
 
-// HTTPRequestMirrorFilter defines configuration for the
-// RequestMirror filter.
+// HTTPRequestMirrorFilter defines configuration for the RequestMirror filter.
```

Comment:
Currently all `HTTPRequestMirrorFilter` fields are optional. Does a validation mechanism exists to ensure at least 1 field contains a value? An empty `HTTPRequestMirrorFilter` doesn't make sense, but I understand why each field is optional.

Reply:
Sadly this will be one more thing that we'll need to rely on a validating webhook for.

Reply:
https://github.com/kubernetes-sigs/service-apis/issues/349